02.replicas.in.sync/new-relic-dashboard.json

```diff
@@ -0,0 +1,260 @@
+{
+    "name": "M&T Troubleshooting",
```

Comment:
curious - what's M&T?

Reply:
Monitoring & Troubleshooting. Now that you mention it, two troubleshootings don't quite work :) . I'll fix it.